Source/items.cpp

```diff
@@ -2571,13 +2571,32 @@ void CalcPlrItemVals(Player &player, bool loadgfx)
 	int imana = 0; // increased mana
 
 	int spllvladd = 0; // increased spell level
-	int enac = 0;      // enhanced accuracy
+	int enac = 0;      // enhanced armor class

```

Comment:
wrong

Reply:
![image](https://user-images.githubusercontent.com/14297035/153097146-db8abbd5-6abc-4adc-9700-9e3fa5cac79c.png)
you even changed it in your other PR xD